### PR TITLE
SCRS-11003 Passing SIC Codes to ICL

### PR DESCRIPTION
**Pull request by Daveo-G**
# SCRS-11003 SIC code look up from Companies House

 New feature

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs and context.
Passes SIC codes to pre-pop ICL
## Checklist

* [X] I've included appropriate tests with any code I've added
* [X] I've executed the acceptance test pack locally to ensure there are no regressions
* [X] I've added my code using logical, atomic commits, squashing as appropriate - including the Jira issue number in the commit message
* [X] I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

**Comment by ChrisHillCap** [comment]
before merging do error message speak to me

**Comment by CEPoole** [comment]
image
@ChrisHillCap are you suggesting this needs to be part of the dynamic content?

File: test/services/IncorporationInformationServiceSpec.scala [hunk 1]
```diff
@@ -98,4 +100,85 @@ class IncorporationInformationServiceSpec extends VatRegSpec with Inspectors wit
         testIncorporationInfo.statusEvent.incorporationDate
     }
   }
+
+  "retrieveSicCodes" must {
```
Comment (ChrisHillCap) [inline test/services/IncorporationInformationServiceSpec.scala]: would like to see a test where the response = 200, json is invalid, does this recover correctly?

**Comment by ChrisHillCap** [comment]
This looks ready to merge just squash before merging

**Pull request merged**